operator/gitops/argocd/logging-operator/olo-namespace.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-logging
```

Comment:
Ditto - link to docs. https://docs.openshift.com/container-platform/4.11/logging/cluster-logging-deploying.html#cluster-logging-deploy-cli_cluster-logging-deploying